composer.json

```diff
@@ -15,28 +15,28 @@
         }
     ],
     "require": {
-        "php": "^7.2.5",
+        "php": "^7.3|^8.0",
         "ext-json": "*",
         "ext-openssl": "*",
         "composer/semver": "^3.0",
-        "guzzlehttp/guzzle": "^6.3.1",
-        "illuminate/http": "^7.0",
-        "illuminate/validation": "^7.18",
+        "guzzlehttp/guzzle": "^6.5.5|^7.0",
+        "illuminate/http": "^8.0",
+        "illuminate/validation": "^8.0",
         "laminas/laminas-text": "^2.7",
-        "laravel-zero/framework": "^7.0",
+        "laravel-zero/framework": "^8.0",
         "m1/env": "^2.2",
         "pragmarx/yaml": "^1.2",
         "symfony/config": "^5.0",
-        "titasgailius/terminal": "^0.4.0",
+        "titasgailius/terminal": "^1.0",
         "tmarois/filebase": "^1.0"
     },
     "require-dev": {
         "bamarni/symfony-console-autocomplete": "^1.3",
         "fzaninotto/faker": "^1.4",
         "jaschilz/php-coverage-badger": "^2.0",
-        "mockery/mockery": "^1.3.1",
+        "mockery/mockery": "dev-master",
```

Comment:
This contains a bug fix not yet released. I'll update in the future once that release comes out.